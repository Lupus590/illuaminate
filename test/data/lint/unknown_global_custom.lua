-- config: (lint (only var:unknown-global) (globals :lua5.3 foo))

-- foo is a custom global so is ok
print(foo)

-- getfenv is not present in Lua 5.3, so produces a warning.
print(getfenv)
