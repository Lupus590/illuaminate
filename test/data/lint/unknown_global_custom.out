unknown_global_custom.lua:[7:7-7:13]: Using unknown global "getfenv" [var:unknown-global]
   │
 7 │ print(getfenv)
   │       ^^^^^^^
No errors and 1 warnings
